index.html

```diff
@@ -194,7 +194,7 @@
                   <a href="http://www.razerzone.com/store/razer-deathstalker-chroma" target="_blank">
                     <img src="https://assets.razerzone.com/eeimages/products/22563/rzr_deathstalker_chroma_03__store_gallery.png">
                     <br>
-                    <h5>DeathStalker Chroma</h5>
+                    <h5>Deathstalker Chroma</h5>
```

Comment:
You're right, thanks for the spot -- I was basing it on what Razer had in the title bar for a different DeathStalker:

![image](https://cloud.githubusercontent.com/assets/13032135/22628038/9b22d984-ebc4-11e6-86c0-4a77439e886c.png)

 :confused: 

![image](https://cloud.githubusercontent.com/assets/13032135/22628017/4987b5fe-ebc4-11e6-910d-6a92e47786a5.png)